sys/include/net/gcoap.h

```diff
@@ -231,6 +231,11 @@
 extern "C" {
 #endif
 
+/**
+ * @defgroup net_gcoap_conf    Gcoap compile configurations
+ * @ingroup  config
```

Comment:
`@ingroup   net_gcoap` missing?

Reply:
Sorry, I omitted this tag on all three PRs. Will fix.